docs/ascat.md

```diff
@@ -23,7 +23,7 @@ LogRi(tumor)=log2((countsAi(tumor)+countsBi(tumor))/(countsAi(normal)+countsBi(n
 LogRi(normal)=0
 ```
 
-For male samples, the X chromosome markers have special treatment:
+For male samples, the X and Y chromosome markers have special treatment:
```

Comment:
Not sure about that actually.
It would indeed make sense, but I'll better mention @malinlarsson or @szilvajuhos if they know the answer here